mirobo/ceil_cli.py

```diff
@@ -97,9 +97,9 @@ def status(dev: mirobo.Ceil):
         return  # bail out
 
     click.echo(click.style("Power: %s" % res.power, bold=True))
-    click.echo("Brightness: %s" % res.bright)
-    click.echo("CCT: %s" % res.cct)
-    click.echo("Scene Number: %s" % res.snm)
+    click.echo("Brightness: %s" % res.brightness)
+    click.echo("Correlated color temperatur: %s" % res.color_temperature)
+    click.echo("Scene: %s" % res.scene)
     click.echo("dv: %s" % res.dv)
     click.echo("Smart Midnight Light: %s" % res.bl)
     click.echo("Auto CCT: %s" % res.ac)
```

Comment:
Could you also rename `ac` and `bl`, maybe `auto(matic)_color_temperature` and `smart_midnight_mode` or similar.